pkg/codegen/java/testdata/TestFormatForeignComments/repro_auth0_failure.golden

```diff
@@ -786,11 +786,11 @@
  *                 .protocolBinding("urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST")
  *                 .requestTemplate("""
  * \u003csamlp:AuthnRequest xmlns:samlp="urn:oasis:names:tc:SAML:2.0:protocol"
- * {@literal @@}AssertServiceURLAndDestination{@literal @@}
- *     ID="{@literal @@}ID{@literal @@}"
- *     IssueInstant="{@literal @@}IssueInstant{@literal @@}"
- *     ProtocolBinding="{@literal @@}ProtocolBinding{@literal @@}" Version="2.0"\u003e
- *     \u003csaml:Issuer xmlns:saml="urn:oasis:names:tc:SAML:2.0:assertion"\u003e{@literal @@}Issuer{@literal @@}\u003c/saml:Issuer\u003e
+ * }{@literal @}{@literal @}{@code AssertServiceURLAndDestination}{@literal @}{@literal @}{@code
+ *     ID="}{@literal @}{@literal @}{@code ID}{@literal @}{@literal @}{@code "
+ *     IssueInstant="}{@literal @}{@literal @}{@code IssueInstant}{@literal @}{@literal @}{@code "
+ *     ProtocolBinding="}{@literal @}{@literal @}{@code ProtocolBinding}{@literal @}{@literal @}{@code " Version="2.0"\u003e
+ *     \u003csaml:Issuer xmlns:saml="urn:oasis:names:tc:SAML:2.0:assertion"\u003e}{@literal @}{@literal @}{@code Issuer}{@literal @}{@literal @}{@code \u003c/saml:Issuer\u003e
```

Comment:
Yea, I've left this in since it purports to contain a regression issue, but it's not super nice and actually I don't believe it produces valid Javadoc (as in, the source is fundamentally invalid and always has been).